Rocket.Chat.iOS/Info.plist

```diff
@@ -53,5 +53,10 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>UIAppFonts</key>
+	<array>
+		<string>Roboto-Regular</string>
+		<string>Roboto-Medium</string>
```

Comment:
@kormic @gsaslis I don't know... we can keep as it is for now, but I'm a fan of native font (Now, San Francisco) :-)